### PR TITLE
jgit integration to fetch scala sources

### PR DESCRIPTION
**Pull request by andreaTP**
Targeting #92

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 scalalib/src/
 target/
 *.dot
+image0.ppm
```
Comment (densh) [inline .gitignore]: I don't think that we should have this one in gitignore as it's really demo-specific.

File: docker/Dockerfile [hunk 2]
```diff
@@ -43,14 +43,10 @@ WORKDIR /tmp
 
 RUN git clone https://github.com/scala-native/scala-native.git
 
-WORKDIR /tmp/scala-native/submodules
-
-RUN rm -rf scala && git clone https://github.com/scala/scala.git
-
```
Comment (densh) [inline docker/Dockerfile]: Just a few issues left to address to make it work out-of-the-box on linux. 👍

File: .gitignore (hunk 1 above)
Comment (andreaTP) [inline .gitignore]: ok I'll fix and rebase

File: docker/Dockerfile (hunk 2 above)
Comment (andreaTP) [inline docker/Dockerfile]: Could I work on this?
How do you prefer to target this feature? sbt settings, environment checks, use of external tools (i.e.  cmake) or whatever?

**Comment by andreaTP** [comment]
fixed commit

**Comment by densh** [comment]
LGTM, thanks @andreaTP !

File: docker/Dockerfile (hunk 2 above)
Comment (densh) [inline docker/Dockerfile]: https://github.com/scala-native/scala-native/issues/102

**Pull request merged**